### DISCUSSION TITLE
To paste table data into affine table

### DISCUSSION THREAD

**Discussion by Ding-Fan** [toeverything/AFFiNE#2465] [Ideas]
I'm currently working with table data from Notion, which I've exported to a CSV file and opened in LibreOffice Calc (similar to Excel). I've encountered an issue when attempting to copy this data into an Affine table. It seems that only the first column can be successfully copied and pasted into the first column of the Affine table. How to improve the usability of the table copy-paste feature?

**Comment by HeJiachen-PM** [comment]
This is mainly because you were probably using AFFiNE database table-view and the first column of it is by default the heading column. We are planning to provide support for normal simple tables, but it is not yet finished.
